raphtory-graphql/src/model/filters/property.rs

```diff
@@ -24,6 +27,23 @@ impl PropertyHasFilter {
             None => node.vv.properties().values().any(valid_prop),
         };
     }
+
+    pub(crate) fn matches_edge_properties(&self, edge: &Edge) -> bool {
+        let valid_prop = |prop| valid_prop(prop, &self.value_str, &self.value_num);
+
+        return match &self.key {
+            Some(key) => {
+                if let Some(prop) = raphtory::prelude::EdgeViewOps::properties(&edge.ee).get(key) {
```

Comment:
@Dullaz if you just import raphtory::prelude::* at the top this should be a good bit cleaner and I can merge :)